iiblib/iib_build_details_model.py

```diff
@@ -88,6 +114,9 @@ def _get_args(self, data):
         return attrs
 
     def to_dict(self):
+        """
+        Return dictionary from the object
+        """
```

Comment:
Can we also add `:return: ` and maybe also `:rtype:` as well  so documentation is generated also with it. 
However I can see that it is not that much used in this library.

Reply:
The documentation in iib_build_details_model is added